src/main/interface/cli.c

```diff
@@ -2599,14 +2597,14 @@ static void cliBeeper(char *cmdline)
 void cliRxBind(char *cmdline){
     UNUSED(cmdline);
     switch (rxSpiConfig()->rx_spi_protocol) {
-#ifdef USE_RX_CC2500_BIND
```

Comment:
These should still be conditional to the actual drivers being compiled in.

Reply:
Yup, similar to rx_spi.c, I'll fix that as well.